.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG] "
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Do something unusual to Kubernetes
+2. Make sure there's a specific cluster state present
+3. Run test plan: <link to test plan>
+4. See error: <paste output of error here using GitHub markdown code block styling>
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Test Plan**
+If applicable, add your test plan(s) and link them here.
+
+**Development Environment (please complete the following information):**
+Runner: (if applicable)
+ - OS: 
+ - Run in Docker?: No/Yes(Container Tag)
+- Kubernetes Version: 
```

Comment:
Shouldn't this be aligned with those above?
```suggestion
 - Kubernetes Version: 
```

Reply:
Oh, yes they should! However, this is probably more in line with the rest of the file:
https://github.com/grafana/nethax/pull/56/files#r2093799405